src/app/shared/entity/cluster.ts

```diff
@@ -229,6 +234,11 @@ export enum ProxyMode {
   iptables = 'iptables',
 }
 
+export enum CNIPlugins {
```

Comment:
For enum I'd use singular form, so `CNIPlugin`.

Reply:
I had a problem that it the clashed with `CNIPlugin` class and I thought since the json schema for API has `cniPlugin`, that I should rather conform there.

How about I make the class `CNIPluginConfig`, so the enum can be `CNIPlugin` without the names clashing?

Reply:
I didn't notice the class name. Renaming sounds good but it can also stay as it is. It's minor thing.